.travis.yml

```diff
@@ -90,3 +91,10 @@ script:
   echo;
   echo "Running PHP_CodeSniffer";
   vendor/bin/phpcs --standard=Configuration/PhpCodeSniffer/ bin/ src/ tests/ public/;
+
+after_script:
+- >
+  if [[ $COVERAGE = yes ]]; then
+    wget https://scrutinizer-ci.com/ocular.phar;
```

Comment:
Can we add this to the dev dependencies?
https://packagist.org/packages/scrutinizer/ocular

Reply:
Do we need those packages during local development? I don't see the reason why developer wants to upload their local code coverage results into scrutinizer.

Reply:
We don't need them, but they also don't do any harm. I'd just like to consistently use Composer for downloading dependencies (even the optional ones) instead of the outdated wget.

This also allows the packages to be cached in TravisCI.